examples/3d-tiles/app.js

```diff
@@ -144,13 +142,12 @@ export default class App extends PureComponent {
     });
 
     // The "Additional" examples can contain a coordinate origin
-    const {coordinateOrigin} = tilesetExampleProps;
-    if (coordinateOrigin) {
+    if (tilesetExampleProps && tilesetExampleProps.coordinateOrigin) {
```

Comment:
We should really edit the royalExhibition json so we can get rid of this code. @loshjawrence I think I saw this on your list.

Reply:
Yea, talked to Sean and he mentioned it could just be retiled with a specified lat long.

Reply:
The hard coded `lat, lng` for royalExhibition in the `examples/3d-tiles/app.js` is added by me and it is not accurate.

Reply:
Do we need to retile? Can we just hand edit the transform in the tileset.json?

Reply:
Sorry by re-tile just to figure out what the transform is (rotation and translation). The transform that looked correct in cesium was:

-0.5739749660092391,-0.8188728462922024,0.0,0.0,-0.5019514247607851,0.3518342967041723,0.7900996107094966,0.0,-0.6469911170760465,0.4534973972008964,-0.6129784703859522,0.0,-4131756.7389703253,2896175.013126114,-3888472.663125889,1.0

I'm going to test this in loaders first before I open the PR for updating the json